.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```diff
@@ -0,0 +1,112 @@
+name: Bug Report
+description: File a bug report.
+title: "[Bug]: "
+labels: ["bug", "ready for refinement"]
+projects: ["octo-org/1", "octo-org/44"]
+assignees:
+  - octocat
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details (optional)
+      description: How can we get in touch with you if we need more info?
+      placeholder: ex. email@example.com
+    validations:
+      required: false
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Add as much details about the bug as you can.
+      placeholder: Tell us what happend!
+    validations:
+      required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version
+      description: What version of our F5 NGINX Ingress Controller are you running?
+      options:
+        - edge
+        - 3.6.2
```

Comment:
not for this PR, but it would be nice to automatically append the latest version to this list of versions when we release a new version

Reply:
Good point. I was wondering if there would be some way to store supported version in another file and have the `options` field reference that file. I couldn't find any existing documentation that mentions support for that.